.github/workflows/ci-testing-deploy.yml

```diff
@@ -64,28 +64,27 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: setup docker buildx
-        if: github.event_name == 'push'
         id: buildx
         uses: docker/setup-buildx-action@v2
         with:
           version: ${{ matrix.docker_buildx }}
-          driver: docker
+          driver: docker-container
       - name: setup docker-compose
-        if: github.event_name == 'push'
         run: sudo ./ci/github/helpers/setup_docker_compose.bash ${{ matrix.docker_compose }} ${{ matrix.docker_compose_sha }}
+      - name: expose github runtime for buildx
+        uses: crazy-max/ghaction-github-runtime@v2
       - name: show system environs
-        if: github.event_name == 'push'
         run: ./ci/helpers/show_system_versions.bash
       - name: build images
-        if: github.event_name == 'push'
-        run: ./ci/build/test-images.bash build_images
-      - name: set owner variable
-        if: github.event_name == 'push'
-        run: echo "OWNER=${GITHUB_REPOSITORY%/*}" >> $GITHUB_ENV
-      - name: push images
-        # only pushes have access to the docker credentials
-        if: github.event_name == 'push'
-        run: ./ci/deploy/dockerhub-test-images.bash
+        run: |
+          export DOCKER_IMAGE_TAG=$(exec ci/helpers/build_docker_image_tag.bash)
+          mkdir --parents /tmp/build
```

Comment:
use ``RUNNER_TEMP`` instead of ``/tmp/`` ??


Look for that variable in https://docs.github.com/en/actions/learn-github-actions/environment-variables

Reply:
good point, thanks for the link